### DISCUSSION TITLE
is there anywhere to get the access log? (Audit)

### DISCUSSION THREAD

**Discussion by hhmmll** [Budibase/budibase#2149] [Ideas]
We're building our internal tool using budibase, for the audit requirement, we need to be able to get the access log (ex: what account login/access what application
does budibase support the access log?

**Comment by mjashanks** [comment]
Hey @hhmmll . We don't support that at the moment - although it's something we know we will need.

It will be a feature that appears on our enterprise (paid) plan in the future. We have not started scoping it yet, would love to understand more about what you need.
- Who would access the log (i.e. Role)
- How would that person interact with the audit? E.g. what information would they expect to pull out and why.
- What kind of events would you expect to see in there
- What kind of retention options would you expect - does this need to be configurable

It'd be a great help to us to understand more of the requirements around this. Thanks!

**Reply by joebudi** [reply]
If you use Auth0 with our new OIDC support, they can/will provide access logs.

https://auth0.com/docs/logs
